change/@ni-nimble-components-b0a59426-c9a2-444a-a9f7-1ccf79dd94b4.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
> It's questionable to me that we would revert a general fix for the sake of the mobile experience. My understanding is that mobile support has been deprioritized, so we should not bother fixing those kinds of issues. However, the workaround being reverted does not seem super important (and we're still tracking the issue), so I'm not strongly opposed to this change.

Further to this, the mobile bug doesn't seem too bad to me. I'm not terribly bothered if the keyboard is dismissed when I interact with something else on the page. But I also agree that it's your team's call which bug is more important.

By the way, I did test the Storybook builds on my iPhone both before and after this change and verified that it fixes the issue.

Reply:
Thanks for verifying the behavior. For the other issue https://github.com/ni/nimble/issues/1422, we will wait until the underlying issue in FAST gets fixed, if not we will find some other way to fix the tabindex behavior without affecting the mobile experience.